server/asset/btc/btc.go

```diff
@@ -81,6 +81,7 @@ type btcNode interface {
 // cache of block data for quick lookups. Backend implements asset.Backend, so
 // provides exported methods for DEX-related blockchain info.
 type Backend struct {
+	ready chan struct{}
```

Comment:
This is a downside of using the Backend subsystem's context to also cancel externally-initiated RPCs on subsystem shutdown.  I didn't want to plumb every method with a context but maybe...